src/main/java/com/std/tothebook/api/repository/MyBookCustomRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.std.tothebook.api.repository;
+
+import com.std.tothebook.api.domain.dto.FindMyBookResponse;
+import com.std.tothebook.api.domain.dto.FindMyBooksResponse;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface MyBookCustomRepository {
+
+    List<FindMyBooksResponse> findMyBookByUserId(long userId);
+
+    Optional<FindMyBookResponse> findSimpleMyBook(long id);
```

Comment:
```suggestion
    Optional<FindMyBookResponse> findMyBook(long id);
```
그저 상세 조회라면 simple을 붙인 이유가 궁금합니다!

Reply:
단건은 simple인 줄 알았습니다 😅 특별한 이유가 없어서 MyBook으로 바꾸겠습니다. 의견 감사합니다.